src/development/metrics.md

```diff
@@ -0,0 +1,153 @@
+# Metrics
+
+This page gather all the measurement metrics, so that we can easily refer to them in the future. It is broken into four parts:
+
+ - The measurements that are common to both verkle and binary trees
+ - The measurements that are specific to verkle trees
+ - The measurements that are specific to binary trees
+ - The measurements that are specific to state expiry
+
+This is a living document, that we will fill as questions arise and we gather more data.
+
+##  Common measurements
```

Comment:
@jsign I assume that your "group types" measurements can go in there

Reply:
I lean more into having separate pages under a sub-chapter.

If we don't do that, we can end up with an incredibly long page with numbers that make it hard for people to see what they're looking for.

I'd suggest your report is a sub-page, and mine a separate one.